pages/about.js

```diff
@@ -1,30 +1,15 @@
 import Head from "next/head";
-import styled from "styled-components";
-
-const Hero = styled.div`
-  height: 90vh;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  background: #fff;
-`;
-
-const Heading = styled.h1`
-  color: #000;
-  font-size: 10rem;
-  font-weight: 900;
-`;
-
+import { Heading, Section } from "../components/SectionElements";
 export default function About() {
   return (
     <>
       <Head>
         <title>About</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <Hero>
```

Comment:
maybe we can think about layout instead of copying and pastying every page. Example here: https://nextjs.org/docs/basic-features/layouts